metadata.yaml

```diff
@@ -1,19 +1,31 @@
 # Copyright 2021 Unicorn
 # See LICENSE file for licensing details.
-name: prometheus-bind-exporter-operator
+name: prometheus-bind-exporter
 description: |
-  TEMPLATE-TODO: fill out the charm's description
+  The subordinate charm for the designate-bind charm, which provide the
+  prometheus-bind-exporter snap to collect metrics about BIND v9+. At the same
+  time, it provides an opportunity to create a relation with prometheus2 and
+  grafana charm.
 summary: |
-  TEMPLATE-TODO: fill out the charm's summary
+  Export BIND (named/dns) v9+ service metrics to Prometheus.
 
-# TEMPLATE-TODO: replace with containers for your workload (delete for non-k8s)
-containers:
-  httpbin:
-    resource: httpbin-image
+subordinate: true
 
-# TEMPLATE-TODO: each container defined above must specify an oci-image resource
 resources:
-  httpbin-image:
-    type: oci-image
-    description: OCI image for httpbin (kennethreitz/httpbin)
+  prometheus-bind-exporter:
+    type: file
+    filename: promethues-bind.exporter.snap
 
+requires:
+  juju-info:
```

Comment:
This is just a suggestion but I recently also made a new subordinate charm taht's meant to be deployed on `kubernetes-master` and I named the subordinate relation `kubernetes` just to give a bit of a context to the user